internal/storage/bucket_handle.go

```diff
@@ -64,15 +65,17 @@ func (bh *bucketHandle) BucketType() gcs.BucketType {
 			bh.bucketType = gcs.NonHierarchical
 			return bh.bucketType
 		}
-
+		startTime := time.Now()
+		logger.Infof("\"GetStorageLayout <- (%s)\"", bh.bucketName)
```

Comment:
Why do we need the quotes around the log message ? Isn't that automatically added by the logger ?

Reply:
Removed.